network/core-node/admin-guide/monitoring.mdx

```diff
@@ -1,93 +1,89 @@
 
-You can access this information using commands and inspecting Stellar Core's output, which is what the first half of this doc covers. You can also connect [Prometheus](#using-prometheus) to make monitoring easier, combine it with [Alertmanager](#configure-notifications-using-alertmanager) to automate notification, and use pre-built [Grafana dashboards](#visualize-metrics-using-grafana) to create visual representations of your node's well-being.
+You can access this information using commands and inspecting Stellar Core's output. The first half of this page will cover this approach. You can also connect [Prometheus](#using-prometheus) to make monitoring easier, combine it with [Alertmanager](#configure-notifications-using-alertmanager) to automate notification, and use pre-built [Grafana dashboards](#visualize-metrics-using-grafana) to create visual representations of your node's well-being.
 
 However you decide to monitor, the most important thing is that you have a system in place to ensure that your integration keeps ticking.
 
 ## General Node Information
 
 If you run `$ stellar-core http-command 'info'`, the output will look something like this:
 
-<CodeExample>
-
 ```json
 {
-      "build" : "v11.1.0",
-      "history_failure_rate" : "0",
-      "ledger" : {
-         "age" : 3,
-         "baseFee" : 100,
-         "baseReserve" : 5000000,
-         "closeTime" : 1560350852,
-         "hash" : "40d884f6eb105da56bea518513ba9c5cda9a4e45ac824e5eac8f7262c713cc60",
-         "maxTxSetSize" : 1000,
-         "num" : 24311579,
-         "version" : 11
-      },
-      "network" : "Public Global Stellar Network ; September 2015",
-      "peers" : {
-         "authenticated_count" : 5,
-         "pending_count" : 0
-      },
-      "protocol_version" : 10,
-      "quorum" : {
-         "qset" : {
-            "agree" : 6,
-            "delayed" : 0,
-            "disagree" : 0,
-            "fail_at" : 2,
-            "hash" : "d5c247",
-            "ledger" : 24311579,
-            "missing" : 1,
-            "phase" : "EXTERNALIZE"
-         },
-         "transitive" : {
-            "critical" : null,
-            "intersection" : true,
-            "last_check_ledger" : 24311536,
-            "node_count" : 21
-         }
-      },
-      "startedOn" : "2019-06-10T17:40:29Z",
-      "state" : "Catching up",
-      "status" : [ "Catching up: downloading and verifying buckets: 30/30 (100%)" ]
-   }
+  "build": "v11.1.0",
+  "ledger": {
+    "age": 3,
+    "baseFee": 100,
+    "baseReserve": 5000000,
+    "closeTime": 1560350852,
+    "hash": "40d884f6eb105da56bea518513ba9c5cda9a4e45ac824e5eac8f7262c713cc60",
+    "maxTxSetSize": 1000,
+    "num": 24311579,
+    "version": 11
+  },
+  "network": "Public Global Stellar Network ; September 2015",
+  "peers": {
+    "authenticated_count": 5,
+    "pending_count": 0
+  },
+  "protocol_version": 10,
```

Comment:
How about getting a new one since we are at Protocol 20 in mainnet. Here is an example of the latest from master branch that speaks P21 but is closing ledger version 20, https://gist.github.com/anupsdf/7fb4e2f5b26a0fe04a705ac3e8d032b5

Reply:
Feel free to run from your node (quickstart if you have one running). Will be nice to show state of `Synced` in the bottom.

Reply:
really great suggestion!